delta/src/main/resources/app.conf

```diff
@@ -40,11 +40,12 @@ app {
     default-timeout = 30 seconds
   }
 
-  # Service account settings
-  service-account {
-    # the service account token, if needed (when not required, anonymous subject will be used)
-    token = null
-  }
+  # caller settings for when the service wants perform service calls. Do not asign a value if you want the caller to be anonymous
```

Comment:
To delete or to uncomment I guess

Reply:
No. It is left as a documented example of which parameters needs to be set when defining a caller